### PR TITLE
Updating Nuget package for OData Client to remove NetFx45 binaries

### PR DESCRIPTION
**Pull request by AlanWong-MS**
### Issues
Issue #731

### Description
- Updating Nuget package for OData Client to remove NetFx45 binaries.
- Fixing bug in ServiceWrapperApp where exception is ignored.

### Checklist (Uncheck if it is not completed)

- [x] *Test cases added (N/A)*
- [x] *Build and test with one-click build and test script passed*

### Additional work necessary
N/A

### CODE REVIEW

File: test/EndToEndTests/Services/ServiceWrapperApp/ServiceWrapperApp.cs [hunk 1]
```diff
@@ -67,6 +67,8 @@ static int Main(string[] args)
             {
                 Console.WriteLine(e.Message);
                 PrintHelp();
+
+                return 1;
```
Comment (robward-ms) [inline test/EndToEndTests/Services/ServiceWrapperApp/ServiceWrapperApp.cs]: How about make this 2 so you have a unique return code here, might be useful later on the calling side?

Comment (robward-ms) [reply test/EndToEndTests/Services/ServiceWrapperApp/ServiceWrapperApp.cs]: Your change certainly covers the case I asked for :)

Comment (AlanWong-MS) [reply test/EndToEndTests/Services/ServiceWrapperApp/ServiceWrapperApp.cs]: Wanted to polish the code a little bit more while I was in there. :)

**Comment by AlanWong-MS** [comment]
Looks like the buddy build failed for a separate reason. Will squash and merge this PR.

**Pull request merged**